src/GraphQL.SystemTextJson/StringExtensions.cs

```diff
@@ -0,0 +1,48 @@
+using System.Collections.Generic;
+using System.Text.Json;
+
+namespace GraphQL.SystemTextJson
+{
+    public static class StringExtensions
+    {
+        private static readonly JsonSerializerOptions _jsonOptions = new JsonSerializerOptions()
```

Comment:
```suggestion
        private static readonly JsonSerializerOptions _jsonOptions = new JsonSerializerOptions
```

Reply:
Lol, I'll never prefer the short hand, I want to see the ctor call! But I'll conform :P